server/store/adapters/rdbms/upgrade_fixes.go

```diff
@@ -429,6 +430,34 @@ func fix_2022_09_00_addMissingNodeIdOnFederationMapping(ctx context.Context, s *
 	)
 }
 
+func fix_2022_09_07_changePostgresIdColumnsDatatype(ctx context.Context, s *Store) (err error) {
+	var tableName string
+	if s.DB.DriverName() != "postgres" {
+		return
+	}
+
+	rows, err := s.DB.QueryContext(ctx, "SELECT table_name FROM INFORMATION_SCHEMA.COLUMNS  WHERE column_name = 'id'")
```

Comment:
Consider somehow constraining these ID fields only to the tables that Corteza creates.
A potential edge case is when someone would add some of their own tables for DAL and so forth.